### PR TITLE
supporting PERCY_DO_N0T_CAPTURE_RESPONSIVE_ASSETS env variable to disable fetching of responsive assets

### PR DESCRIPTION
**Pull request by prklm10**
supporting PERCY_DO_N0T_CAPTURE_RESPONSIVE_ASSETS env variable to disable fetching of responsive assets

### CODE REVIEW

File: packages/core/src/percy.js [hunk 1]
```diff
@@ -164,7 +164,11 @@ export class Percy {
       if (!this.skipDiscovery) yield this.#discovery.start();
       // start a local API server for SDK communication
       if (this.server) yield this.server.listen();
-      if (this.projectType === 'web') this.deviceDetails = yield this.client.getDeviceDetails(this.build?.id);
+      if (this.projectType === 'web') {
+        if (!process.env.PERCY_DISABLE_DEVICE_DETAIL_FETCH || process.env.PERCY_DISABLE_DEVICE_DETAIL_FETCH !== 'true') {
```
Comment (ninadbstack) [inline packages/core/src/percy.js:168]: Not a very easy to understand flag for someone who doesnt know percy internals. Why should someone enable/disable it, what does it do ? 

use a better flag name, that allows user to toggle a functionality and not an input to a functionality

Comment (prklm10) [reply packages/core/src/percy.js]: renamed to `PERCY_DON0T_CAPTURE_RESPONSIVE_ASSETS`

Comment (ninadbstack) [reply packages/core/src/percy.js]: Let's make it DO_NOT as those are separate words.

**Review by ninadbstack — APPROVED** [review]

File: packages/core/src/percy.js (hunk 1 above)
Comment (prklm10) [reply packages/core/src/percy.js]: updated

**Review by ninadbstack — APPROVED** [review]

**Pull request merged**